test/buildkitetestjson.jl

```diff
@@ -123,13 +123,31 @@ end
 
 function collect_results!(results::Vector{Dict{String, Any}}, testset::Test.DefaultTestSet, prefix::String="")
     common_data = result_dict(testset, prefix)
+    result_offset = length(results) + 1
+    result_counts = Dict{UInt64, Int}()
```

Comment:
Could just use `Tuple{String, String}` instead of using `hash`

Reply:
Mmmm, why not.